scripts/setup.py

```diff
@@ -148,7 +148,17 @@ def main(args=None):
     # as $PYTHONPATH).
     create_directory(common.OPPIA_TOOLS_DIR)
     create_directory(common.THIRD_PARTY_DIR)
+    common.create_readme(
+        common.THIRD_PARTY_DIR,
+        'This folder contains third party libraries used in Oppia codebase.\n'
+        'You can remove this folder by deleting it and add it '
```

Comment:
"remove" --> "regenerate"
"add it again by running" --> "then running"

(here and below)

Reply:
Updated.